test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs

```diff
@@ -25,25 +25,21 @@ public Task Build_SimpleMvc_UsingDesktopMSBuild_CanBuildSuccessfully()
 
         private async Task Build_SimpleMvc_CanBuildSuccessfully(MSBuildProcessKind msBuildProcessKind)
         {
-            var result = await DotnetMSBuild("Build", "/p:RazorCompileOnBuild=true", msBuildProcessKind: msBuildProcessKind);
+            var result = await DotnetMSBuild("Build", "/p:RazorCompileOnBuild=true /t:_IntrospectPreserveCompilationContext", msBuildProcessKind: msBuildProcessKind);
```

Comment:
Let's not overload these tests. 

Can we instead add verification to this test that the compilation context is produced and then verify 1-2 negative cases (library, no razor files)?

Reply:
Sure